OpenQA/Qemu/Proc.pm

```diff
@@ -233,7 +236,8 @@ sub configure_blockdevs {
     my $is_first = 1;
     for my $k (sort grep { /^ISO_\d+$/ } keys %$vars) {
         my $addoniso = $vars->{$k};
-        my $i        = $k;
+        $addoniso = File::Spec->rel2abs($addoniso) unless File::Spec->file_name_is_absolute($addoniso);
+        my $i = $k;
```

Comment:
This is a good opportunity to increase code coverage then :)

Reply:
Added